plugins/Karma/plugin.py

```diff
@@ -315,7 +315,10 @@ def doPrivmsg(self, irc, msg):
                self.registryValue('allowUnaddressedKarma',
                                   msg.channel, irc.network):
                 irc = callbacks.SimpleProxy(irc, msg)
-                thing = msg.args[1].rstrip()
+                inc = self.registryValue('incrementChars', msg.channel, irc.network)
+                dec = self.registryValue('decrementChars', msg.channel, irc.network)
+                if msg.args[1].endswith(inc) or msg.args[1].endswith(dec):
+                    thing = msg.args[1].rstrip(":, ")
```

Comment:
I don't think this will actually work - if the message ends with `++` or `--` as in the default configuration, there is no way it'll also end with one of `:, `, so this is effectively a no-op.

I think this code should belong in either `_doKarma` or `_normalizeThing` so the behaviour is consistent with addressed Karma calls.